lib/Rules/Lowercase.php

```diff
@@ -30,8 +30,13 @@ class Lowercase extends Base {
 	 */
 	public function verify($password, $val) {
 		if ($this->countLowercase($password) < $val) {
-			throw new PolicyException(
-				$this->l10n->t('The password contains too few lowercase characters. At least %d lowercase characters are required.', [$val]));
+			if ($val > 1) {
```

Comment:
no - please use n() - this function was design to respect different language counting schemas.
Thanks for noticing - we always forget about this problem